internal/fs/fs.go

```diff
@@ -121,12 +121,15 @@ func NewFileSystem(
 
 	mtimeClock := timeutil.RealClock()
 
+	// TODO ezl Consider setting contentcache to nil when the local file cache is disabled
+	// This means we need to refactor and separate the local file cache logic away from
+	// temp files being used when users want to write/modify files in the filesystem
 	contentCache := contentcache.New(cfg.TempDir, mtimeClock)
 
 	if cfg.LocalFileCache {
 		err := contentCache.RecoverCache()
 		if err != nil {
-			fmt.Printf("Encountered error retrieving files from cache directory, disabling local file cache: %v", err)
+			fmt.Printf("encountered error retrieving files from cache directory, disabling local file cache: %v", err)
```

Comment:
Logs (`fmt.Printf`) should start with capitalized letter whereas error strings (`fmt.Errorf`) should not.